WDAC-Policy-Wizard/app/src/MainForm.cs

```diff
@@ -1095,8 +1095,28 @@ void SetAdditionalParameters(BackgroundWorker worker)
                 // If supplemental policy, set the Base policy guid; reset the policy ID and set the policy name
                 else if (this.Policy._PolicyType == WDAC_Policy.PolicyType.SupplementalPolicy)
                 {
-                    // Set BasePolicyID to match the Id of the linked base policy
-                    siPolicy.BasePolicyID = Helper.DeserializeXMLtoPolicy(this.Policy.BaseToSupplementPath).BasePolicyID;
+                    // User provided path to base policy XML
+                    if(this.Policy.BasePolicyId == Guid.Empty)
+                    {
+                        // Set BasePolicyID to match the Id of the linked base policy
+                        siPolicy.BasePolicyID = Helper.DeserializeXMLtoPolicy(this.Policy.BaseToSupplementPath).BasePolicyID;
+                    }
+
+                    // User provided the GUID of the base policy only
+                    else
+                    {
+                        string baseId = this.Policy.BasePolicyId.ToString().ToUpper();
+
+                        // Set BasePolicyID to match the Id provided base policy ID
+                        if (baseId.Contains("{"))
+                        {
+                            siPolicy.BasePolicyID = baseId; 
+                        }
+                        else
+                        {
+                            siPolicy.BasePolicyID = "{" + baseId + "}";
```

Comment:
may have access to this https://learn.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=net-7.0#system-guid-tostring(system-string)

Reply:
I desperately need to update my .net version

Reply:
available as far back as [`.NET Framework 4.7.2`](https://learn.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=netframework-4.7.2#system-guid-tostring(system-string)), at least.

Reply:
SiPolicy.BasePolicy and .PolicyId are type string anyway so I need to convert at sometime